File: Module.Frontend.TPM/Client/model/tpm/rollingvolume/RollingVolume.js
Ext.define('App.model.tpm.rollingvolume.RollingVolume', {
    extend: 'Ext.data.Model',
    idProperty: 'Id',
    breezeEntityType: 'RollingVolume',
    fields: [
        { name: 'Id', hidden: true },
        { name: 'ProductId', useNull: true, hidden: true, isDefault: true },
        { name: 'DemandGroup', useNull: true, type: 'string', hidden: false, isDefault: true },
        { name: 'DMDGroup', useNull: true, type: 'string', hidden: false, isDefault: false },
        { name: 'Week', useNull: true, type: 'string', hidden: false, isDefault: true },
        { name: 'PlanProductIncrementalQTY', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'Actuals', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'OpenOrders', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'ActualOO', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'Baseline', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'ActualIncremental', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'PreliminaryRollingVolumes', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'PreviousRollingVolumes', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'PromoDifference', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'RollingVolumesCorrection', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'RollingVolumesTotal', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'ManualRollingTotalVolumes', useNull: true, type: 'float', hidden: false, isDefault: true },
        { name: 'ZREP', type: 'string',  mapping: 'Product.ZREP', isDefault: true, defaultFilterConfig: { valueField: 'ZREP' }, breezeEntityType: 'Product', hidden: false},
        { name: 'SKU', type: 'string', mapping: 'Product.ProductEN', isDefault: true, defaultFilterConfig: { valueField: 'ProductEN' }, breezeEntityType: 'Product', hidden: false},
        { name: 'BrandTech', type: 'string', mapping: 'Product.BrandsegTechsub', isDefault: true, defaultFilterConfig: { valueField: 'BrandsegTechsub' }, breezeEntityType: 'BrandTech', hidden: false},
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'RollingVolumes',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/TPM/RollingVolume.cs
using Core.Data;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Module.Persist.TPM.Model.TPM
{
    public class RollingVolume : IEntity<Guid>
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }

        public string DemandGroup { get; set; }
        public string DMDGroup { get; set; }
        public string Week { get; set; }
        public double? PlanProductIncrementalQTY { get; set; }
        public double? Actuals { get; set; }
        public double? OpenOrders { get; set; }
        public double? ActualOO { get; set; }
        public double? Baseline { get; set; }
        public double? ActualIncremental { get; set; }
        public double? PreliminaryRollingVolumes { get; set; }
        public double? PreviousRollingVolumes { get; set; }
        public double? PromoDifference { get; set; }

        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
        public double? RollingVolumesCorrection { get; set; }
        public double? RollingVolumesTotal { get; set; }
        public double? ManualRollingTotalVolumes { get; set; } 
        public bool Lock { get; set; }
        public double? FullWeekDiff { get; set; }

        public Guid ProductId { get; set; }
        public virtual Product Product { get; set; }

        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
        public DateTimeOffset? ModifiedDate { get; set; }
    }
}
